isodatetime/tests.py

```diff
@@ -602,8 +605,9 @@ def get_timepointparser_tests(allow_only_basic=False,
                 for time_zone_expr, time_zone_info in time_zone_items:
                     tz_expr = combo_expr + time_zone_expr
                     tz_info = {}
-                    for key, value in (combo_info.items() +
-                                       time_zone_info.items()):
+                    for key, value in \
+                        list(combo_info.items()) + \
+                            list(time_zone_info.items()):
```

Comment:
Maybe use `chain` as well?

Reply:
Really need to get used to `chain` and automatically recognize when it's necessary. Fixed! Thanks!